clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesManifestSpinnakerRelationships.java

```diff
@@ -17,13 +17,11 @@
 
 package com.netflix.spinnaker.clouddriver.kubernetes.v2.description;
 
-import lombok.Builder;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
-@Builder
```

Comment:
FYI @duftler this was removed because it was causing problems with serialization. Jackson requires either an empty or annotated constructor which this was overriding. When I supplied my own, it overwrote the builder's constructor causing it to fail.